### ISSUE TITLE
Return UiImage to pass it to new ViewController.

### ISSUE THREAD

**Issue by Emze100** [Awalz/SwiftyCam#248]
How do I pass the taken image into another ViewController, which is pushed onto the CameraViewController. Do I have to use the didTakePhoto Delegate?

**Comment by angeldzzz23** [comment]
Yes you would have to do and  you can easily pass the photo to another view controller using the given UIImage. 


![Screen Shot 2022-05-26 at 9 41 58 PM](https://user-images.githubusercontent.com/29695936/170630537-56244675-5180-4936-9517-3ea525b82cbf.png)
